Libraries/LibGfx/Font/TypefaceSkia.cpp

```diff
@@ -56,6 +56,36 @@ ErrorOr<NonnullRefPtr<TypefaceSkia>> TypefaceSkia::load_from_buffer(AK::Readonly
     return adopt_ref(*new TypefaceSkia { make<TypefaceSkia::Impl>(skia_typeface), buffer, ttc_index });
 }
 
+RefPtr<TypefaceSkia> TypefaceSkia::clone_with_variations(Vector<FontVariationAxis> const& axes) const
+{
+    if (axes.is_empty())
+        return const_cast<TypefaceSkia*>(this);
```

Comment:
could we return `RefPtr<TypefaceSkia const>` from this function instead of using `const_cast` here?